contracts/minter-suite/Minters/MinterSEAV0.sol

```diff
@@ -450,30 +450,32 @@ contract MinterSEAV0 is ReentrancyGuard, MinterBase, IFilteredMinterSEAV0 {
 
     /**
      * @notice Settles any complete auction for token `_settleTokenId` (if
-     * applicable), then initializes an auction for token `_initializeTokenId`
-     * with bid amount and bidder address equal to `msg.value` and
-     * `msg.sender`, respectively.
+     * applicable), then attempts to create a bid for token
+     * `_initializeTokenId` with bid amount and bidder address equal to
+     * `msg.value` and `msg.sender`, respectively.
+     * Intended to gracefully handle the case where a user is front-run by
+     * one or more transactions to settle and/or initialize a new auction.
```

Comment:
How exactly does this more gracefully handle that case? In the case of front run, one's bid is still going to be too low even if it is an "initialize or bid" type logic vs. just "bid" and "initialize" being separate.

If this doesn't actually more gracefully handle the case, are the changes in this PR still preferable?

Reply:
I've added some additional language here to help clarify.

Regarding handling front-runs, this PR doesn't actually change behaviors of the minter. In either this implementation or in #504, if a user's initial bid is front-run by a different bid, and the user's bid is not sufficiently higher than the other bid, the tx will revert. They also both gracefully handle the case where a collector wants to initialize a new auction with a bid at e.g. 3x initial bid value, and someone else front-runs an initial bid equal to initial bid value. In that case, both implementations place a new bid at 3x the initial bid value, and return funds to the front-runner.

The benefits of this PR are entirely about simplifying the interface to be "settle", "bid", and "settle&bid", removing the concept of "initialize" (since initializing is performed by bidding anyway).

Reply:
That makes sense to me ser – appreciate the additional clarification here re: rationale.